src/dali_backend.cc

```diff
@@ -145,9 +192,12 @@ class DaliModelInstance : public ::triton::backend::BackendModelInstance {
       DaliModel* model, TRITONBACKEND_ModelInstance* triton_model_instance)
       : BackendModelInstance(model, triton_model_instance), dali_model_(model)
   {
-    dali_executor_ = std::make_unique<DaliExecutor>(
-        dali_model_->GetModelProvider().GetModel(), dali_model_->MaxBatchSize(),
-        device_id_);
+    auto serialized_pipeline = dali_model_->GetModelProvider().GetModel();
+    auto max_batch_size = dali_model_->MaxBatchSize();
+    auto num_threads = dali_model_->GetModelParamters().GetNumThreads();
+    DaliPipeline pipeline(serialized_pipeline, max_batch_size,
+                          num_threads, device_id_);
+    dali_executor_ = std::make_unique<DaliExecutor>(std::move(pipeline));
```

Comment:
Is this possible?
```c++
dali_executor_ = std::make_unique<DaliExecutor>({serialized_pipeline, max_batch_size, num_threads, device_id_});
```
Always one less call to `std::move` ;)

Reply:
I would have to write `DaliPipeline{.....}`. It seems too long for me